server/odc-service/src/main/java/com/oceanbase/odc/service/flow/FlowInstanceService.java

```diff
@@ -779,7 +779,12 @@ private void checkCreateFlowInstancePermission(CreateFlowInstanceReq req) {
         if (req.getTaskType() == TaskType.EXPORT) {
             DataTransferConfig parameters = (DataTransferConfig) req.getParameters();
             Map<DBResource, Set<DatabasePermissionType>> resource2Types = new HashMap<>();
-            if (CollectionUtils.isNotEmpty(parameters.getExportDbObjects())) {
+            if (parameters.isExportAllObjects()) {
```

Comment:
Will this mislead users into thinking that exporting the entire database has been successful? I tend to throw exceptions when do not have export dtabase permission. 
What are the opinions of others?

Reply:
If users don't have database export permissions but try to export the whole database, the exception will be thrown. This PR just change the order of judging the `isExportAllObjects` as you could see in the PR description.